example/rds-cluster-tf/variables.tf

```diff
@@ -0,0 +1,45 @@
+variable "profile" {
+  description = "AWS profile to use for authentication"
```

Comment:
I'm not totally clear on what this means. What is AWS profile referring to?

Reply:
The one used in the environment/.configuration file for CLI/and the AWS client library - I wrote the initial DB module that way for configurability and it bubbled up.   In theory this overrides the environment variable, if set.  

We actually don't do this elsewhere and rely *solely* on the environment variables, so that's inconsistent.  I can remove it.

Reply:
7f3ef699b